<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Finance</class>
 <widget class="QWidget" name="Finance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 85, 255);
background-color:rgb(255, 255, 255)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>我的钱！！！！这是我的钱！！！！！！</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Private Savings:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(0, 85, 255);
;</string>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideLeft</enum>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Income</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>135</width>
            <height>152</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Detail:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Account:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Today_Sum:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_6"/>
           </item>
          </layout>
         </widget>
         <widget class="QCommandLinkButton" name="commandLinkButton_2">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>100</y>
            <width>31</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QTreeView" name="treeView_2">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>10</y>
            <width>271</width>
            <height>301</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 85, 255);
background:url(:/Image/sun.png)</string>
          </property>
         </widget>
         <zorder>treeView_2</zorder>
         <zorder>layoutWidget</zorder>
         <zorder>commandLinkButton_2</zorder>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>expend</string>
         </attribute>
         <widget class="QCommandLinkButton" name="commandLinkButton">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>100</y>
            <width>31</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QTreeView" name="treeView">
          <property name="geometry">
           <rect>
            <x>210</x>
            <y>10</y>
            <width>271</width>
            <height>301</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(0, 85, 255);
background:url(:/Image/rain.png)</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>70</y>
            <width>135</width>
            <height>152</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Detail:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Account:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Today_Sum:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
